src/main/kotlin/no/nav/syfo/varsel/VarselService.kt

```diff
@@ -20,60 +20,62 @@ class VarselService(
     private val dokarkivClient: DokarkivClient,
 ) {
     private val log = logger()
-    fun findMerOppfolgingVarselToBeSent(): List<MerOppfolgingVarselDTO> {
-        return varselRepository.fetchMerOppfolgingVarselToBeSent()
+
+    fun findMerOppfolgingVarselToBeSent(): List<MerOppfolgingVarselDTO> =
+        varselRepository
+            .fetchMerOppfolgingVarselToBeSent()
             .filter {
                 pdlClient.isBrukerYngreEnnGittMaxAlder(it.personIdent, 67)
             }
-    }
 
-    fun ferdigstillMerOppfolgingVarsel(
-        fnr: String,
-    ) {
-        val hendelse = ArbeidstakerHendelse(
-            type = HendelseType.SM_MER_VEILEDNING,
-            ferdigstill = true,
-            data = null,
-            arbeidstakerFnr = fnr,
-            orgnummer = null,
-        )
+    fun ferdigstillMerOppfolgingVarsel(fnr: String) {
+        val hendelse =
+            ArbeidstakerHendelse(
+                type = HendelseType.SM_MER_VEILEDNING,
+                ferdigstill = true,
+                data = null,
+                arbeidstakerFnr = fnr,
+                orgnummer = null,
+            )
         producer.sendVarselTilEsyfovarsel(hendelse)
     }
 
     @Suppress("MaxLineLength")
-    fun sendMerOppfolgingVarsel(
-        merOppfolgingVarselDTO: MerOppfolgingVarselDTO,
-    ) {
+    fun sendMerOppfolgingVarsel(merOppfolgingVarselDTO: MerOppfolgingVarselDTO) {
         val personIdent = merOppfolgingVarselDTO.personIdent
         try {
             val pdf = pdfgenService.getMerVeiledningLandingPdf(personIdent)
             val uuid = UUID.randomUUID().toString()
 
-            val dokarkivResponse = dokarkivClient.postDocumentToDokarkiv(
-                fnr = personIdent,
-                pdf = pdf,
-                uuid = uuid,
-            )
-            if (dokarkivResponse != null) {
-                val hendelse = ArbeidstakerHendelse(
-                    type = HendelseType.SM_MER_VEILEDNING,
-                    ferdigstill = false,
-                    data = VarselData(
-                        VarselDataJournalpost(uuid = uuid, id = dokarkivResponse.journalpostId.toString()),
-                        null,
-                        null,
-                        null,
-                    ),
-                    arbeidstakerFnr = personIdent,
-                    orgnummer = null,
+            val dokarkivResponse =
+                dokarkivClient.postDocumentToDokarkiv(
+                    fnr = personIdent,
+                    pdf = pdf,
+                    uuid = uuid,
                 )
+            if (dokarkivResponse != null) {
```

Comment:
Har vi samme formateringsregler? Rart at det er så mange endringer fra din side. Kjører default intellij med Detekt (med Ktlint i detekt enabled)

Reply:
Kan være greit å sjekke at det for kotlin (i settings) er default IDE som format, og at du har detekt pluging satt opp med konfig-fil, at alt behandles som error, og ktlint enabled

Reply:
Mmm, har samme config. Postet configen min i Slack-tråden